server/api.go

```diff
@@ -216,6 +219,11 @@ func MapRouteBySchema(server *Server, dataStore db.DB, s *schema.Schema) {
 			handleError(w, err)
 			return
 		}
+
+		if r.Header.Get("Long-Poll") == "true" {
```

Comment:
using X- is no longer recommended (http://stackoverflow.com/a/3561399) apparently, but sure